# Goal
#
# start: time at which to start the movement (immidiately if 0)
# duration: duration of time the movement should take to complete
# x/y/z/yaw: points defining a movement trajectory
#			 start and end points must be provided
#			 additional points will be used for spline interpolatation
time start
duration duration
geometry_msgs/Vector3[] positions
float64[] yaws
---
# Result
#
# position_final: the final commanded position of the movement
geometry_msgs/Vector3 position_final
float64 yaw_final
---
# Feedback
#
# progress: current progress (%) through the movement (-1 if not started)
# position: current instantaneous position
# velocity: current instantaneous velocity
float64 progress
geometry_msgs/Vector3 position
geometry_msgs/Vector3 velocity
float64 yaw
float64 yawrate
